content/utilities/ms-map-controller/plugin.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * MS Map Controller
+ *
+ * Simple Plugin to pan & zoom the map with keyboard.
+ *
+ * Paning: Arrows (←, ↑, →, ↓)
+ * Zooming: '-' out, '=' in
+ *
+ * Enable logging by setting `window.__MS_MC_DEBUG` to `true` in conosle.
+ *
+ * Author @matthewsimo
+ */
+
+import {
+  html,
+  render,
+} from "https://unpkg.com/htm/preact/standalone.module.js";
+
+class BasePlugin {
+  constructor() {
+    window.__MS_MC_DEBUG = window.__MS_MC_DEBUG || false;
```

Comment:
I assume you define window.__MS_MC_DEBUG and set it true in the console if you want to debug your plugin?
Why do you set it to false if it returns false? Is it not obsolete? Not saying it's wrong or need to be changed, just curious.

Reply:
When the plugin is initialized this syntax will set `window.__MS_MC_DEBUG` to the value if it's been set before in that session, if not, it'll be set to `false`.